README.md

```diff
@@ -0,0 +1,173 @@
+
+OpenRatSLAM is based on the RatSLAM C++ library written by David Ball and Scott Heath, with support and additions by Michael Milford. The RatSLAM algorithm was originally designed and implemented by Michael Milford and Gordon Wyeth.
+
+OpenRatSLAM has only been tested on Ubuntu and as ROS support for platforms other than Linux is still limited, this will probably not change.
+
+* Maintainer status: maintained
+* Maintainer: Peter Rudolph <semael23 AT gmail DOT com>
+* Authors: Michael Milford and Gordon Wyeth (RatSLAM), David Ball and Scott Heath (RatSLAM C++), David Ball (OpenRatSLAM)
+* License: GNU General Public License v3.0
+* Source: git https://github.com/sem23/ratslam.git (branch: ratslam_ros)
+
+## 1. External Documentation
+
+Further documentation can be found on ROS wrapper authors [git](https://github.com/davidmball/ratslam).
+
+## 2. Hardware Requirements
+
+To get RatSLAM working you need a properly setup monocular camera and an odometry source. The odometry can be generated
+using the visual odometry node, which is provided by the package. But note, that a wheel based odometry is much more
+accurate in most cases.
+
+## 3. Example
+
+Several examples are provided [here](https://github.com/davidmball/ratslam/blob/wiki/RatSLAMROS.md#running-ratslam).
+
+## 4. Nodes
+
+### 4.1 ratslam_em
+
+The experience map is used to store the experiences made by the robot. This node provides a topological map, loop closing capabilites
+and a dijkstra pathplanning algorithm.
+
+#### 4.1.1 Subscribed Topics
+
+"odom" ([nav_msgs/Odometry](http://docs.ros.org/api/nav_msgs/html/msg/Odometry.html))
+> The current odometry.
+
+topic_root + "/PoseCell/TopologicalAction" (ratslam_ros/TopologicalAction)
+> The topological action.
+
+topic_root + "/ExperienceMap/SetGoalPose" ([geometry_msgs/PoseStamped](http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html))
+> The goal pose to where path will be planned.
+
+#### 4.1.2 Published Topics
+
+topic_root + "/ExperienceMap/Map" (ratslam_ros/TopologicalMap)
+> The experience map in RatSLAM format.
+
+topic_root + "/ExperienceMap/MapMarker" ([visualization_msgs/Marker](http://docs.ros.org/api/visualization_msgs/html/msg/Marker.html))
+> The experience map as ROS visualization markers for displaying puposes.
+
+topic_root + "/ExperienceMap/RobotPose" ([geometry_msgs/PoseStamped](http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html))
+> The current pose of the robot in map.
+
+topic_root + "/ExperienceMap/PathToGoal" ([nav_msgs/Path](http://docs.ros.org/api/nav_msgs/html/msg/Path.html))
+> The current planned path (published if triggered to plan).
+
+#### 4.1.3 Parameters
+
+**Note:** Only ROS parameters are mentioned. All others are explained in external documentation. The parameter "topic_root" can be found there!
+
+"map_frame" (Default: "map")
+> The name of the map frame.
+
+"odom_frame" (Default: "odom")
+> The name of the odometry frame.
+
+"base_frame" (Default: "base_footprint")
```

Comment:
Hmmm... Wouldn't it be correct for a 2D slam to take the projected pose? And true, base_footprint is an additional transform with base_link as its parent. Nevertheless, I will change it to "base_link" as it is probably more convenient, like you said.